book/moving_around.md

```diff
@@ -24,7 +24,11 @@ In Nushell, we also support double `*` to talk about traversing deeper paths tha
 
 Here, we're looking for any file that ends with ".md", and the two asterisks further say "in any directory starting from here".
 
-In addition to `*`, there is also the `?` pattern which will match a single character. For example, you can match the word "port" by using the pattern `p???`.
+In other shells (like bash), glob expansion happens in the shell and the invoked program (`ls` in the example above) receives a list of matched files. In Nushell however, the string you enter is passed "as is" to the command, and some commands (like `ls`, `mv`, `cp` and `rm`) interpret their input string as a glob pattern. For example the [`ls` command's help page](https://www.nushell.sh/commands/docs/ls.html) shows that it takes the parameter: `pattern: the glob pattern to use (optional)`.
```

Comment:
reading https://github.com/nushell/nushell/issues/9558#issuecomment-1684495547, is it actually a string, or a dedicated type and `help ls` is wrong?